### PR TITLE
[devicelab] measure entire release folder size, zipped

### PR DESCRIPTION
**Pull request by jonahwilliams**
Reland with some defensive fallbacks. if there are spaces in the paths provided to tar, they need to be quoted or else the command hangs

### CODE REVIEW

**Review by cbracken — APPROVED** [review]
LGTM stamp from a Japanese personal seal

File: dev/devicelab/lib/tasks/perf_tests.dart [hunk 1]
```diff
@@ -1513,19 +1513,29 @@ class CompileTest {
...
-        releaseSizeInBytes = exe.lengthSync();
+        // rather a directory containing an .exe and .dll files. Zip them all
+        // together to get an approximate release size.
+        final int result = await exec('tar.exe', <String>['-zcf', 'build/app.tar.gz', '"$buildPath"']);
+        if (result == 0) {
+          final File outputFile = file('build/app.tar.gz');
+          if (outputFile.existsSync()) {
+            releaseSizeInBytes = outputFile.lengthSync();
+          } else {
+            print('does not exist');
```
Comment (cbracken) [inline dev/devicelab/lib/tasks/perf_tests.dart:1532]: Should this be removed?

Comment (jonahwilliams) [reply dev/devicelab/lib/tasks/perf_tests.dart]: Added more details. If this works .. I'll send a second patch to remove these

**Review by cbracken — APPROVED** [review]
moar lgtm

**Pull request merged**